data-release-pipeline/prepare_for_molgenis/setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup, find_packages
```

Comment:
how does this script work on e.g. the gearshift cluster with the PythonPlus module?

Reply:
I think/hope it works out of the box. I'll verify it, and if it doesn't, a virtualenv will fix it. I'll add the required steps that work to the SOP.